ml-agents/mlagents/trainers/ppo/trainer.py

```diff
@@ -99,7 +100,11 @@ def process_trajectory(self, trajectory: Trajectory) -> None:
         )
         for name, v in value_estimates.items():
             agent_buffer_trajectory["{}_value_estimates".format(name)].extend(v)
-            self.stats[self.policy.reward_signals[name].value_name].append(np.mean(v))
+            stats.stats_reporter.add_stat(
+                self.summary_path,
+                self.policy.reward_signals[name].value_name,
+                np.mean(v),
```

Comment:
Is v always non-empty? Do you need to guard against NaNs anywhere?

Reply:
Guess it's same behavior as before...

Reply:
v shouldn't be NaN unless there are NaNs in the network (dun dun dun)